src/UnitTests/test_quote_prov.cpp

```diff
@@ -184,14 +220,32 @@ extern void QuoteProvTests()
     //
     start = std::clock();
     GetCertsTest();
-    duration_local = (std::clock() - start) / (double) CLOCKS_PER_SEC;
+    duration_local = (std::clock() - start) / (double)CLOCKS_PER_SEC;
 
     GetCrlTest();
 
-    // Ensure that there is a signficiant enough difference between the cert fetch
-    // to the end point and cert fetch to local cache and that local cache call is
-    // fast enough
-    assert (fabs(duration_curl - duration_local) > 0.0001 && duration_local < 0.001);
-    
+    // Ensure that there is a signficiant enough difference between the cert
+    // fetch to the end point and cert fetch to local cache and that local cache
+    // call is fast enough
+    assert(fabs(duration_curl - duration_local) > 0.0001 && duration_local < 0.001);
+

```

Comment:
I am wondering why those 0.0001 and 0.001 were pick.
The timing test like this is usually not stable

Reply:
Yea this was my initial attempt at capturing a perf distinction between fetches from the PCK Caching Service and the Local cache. I'm open to other approach for testing out this particular scenario.

Reply:
If the purpose is to make sure the data is actually fetched from local cache, is it possible to do it via checking the delta of the cache contents before  and after test operations instead of the fetched time?

Reply:
Ok we'll plan to address that as part of @msft-chow's upcoming pull request for local cache related changes.